src/webview/utils/window.ts

```diff
@@ -0,0 +1,49 @@
+/********************************************************************************
+ * Copyright (C) 2024 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+export function fontSize(column: HTMLElement): number {
+    return parseFloat(getComputedStyle(column).fontSize);
+}
+
+/**
+ * Calculates the width of the element without any padding / margin / border
+ */
+export function elementInnerWidth(element: HTMLElement): number {
+    const styles = window.getComputedStyle(element);
+    const padding = parseFloat(styles.paddingLeft) + parseFloat(styles.paddingRight);
+    const columnWidth = element.clientWidth - padding;
+    return columnWidth;
+}
+
+/**
+ * Calculates the width of the provided text within the container
+ *
+ * @see https://stackoverflow.com/a/21015393
```

Comment:
In general, it is not good practice to copy Stack Overflow code, as it implicates the including project in the licensing of the included code, which is a copy-left license:

https://stackoverflow.com/help/licensing
https://creativecommons.org/licenses/by-sa/4.0/

Reply:
I updated the link to use the API [doc](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/measureText) as I only provided the link in the beginning to read more about the topic if necessary